src/kms-key.tf

```
@@ -0,0 +1,90 @@
+module "kms_key_archive" {
+  source  = "cloudposse/kms-key/aws"
+  version = "0.12.2"
```

for folks who don't want to kms encrypt this. could we add a variable that sets the `enabled` field for this module?
```suggestion
  version = "0.12.2"

  enabled = var.enable_kms_encryption
```

Also, you'll probably want to output the arn of the kms key